src/dmd/s2ir.d

```diff
@@ -448,17 +445,6 @@ private extern (C++) class S2irVisitor : Visitor
         // At last, we know which try block this label is inside
         label.lblock.Btry = blx.tryblock;
 
-        // Go through the forward references and check.
-        if (label.fwdrefs)
-        {
-            block *b = label.fwdrefs;
-
-            if (b.Btry != label.lblock.Btry)
-            {
-                // Check that lblock is in an enclosing try block
-                label.lblock.checkEnclosedInTry(b, s);
-            }
-        }
```

Comment:
This part has been lost in the transition causing: https://issues.dlang.org/show_bug.cgi?id=20829 and also this compiles now:

```d
void fun()
{
    throw new Exception("haw haw");
}
void main()
{
    goto g;                                                                                                                                 
    writeln("before try");
    try   
    {  
g:
        writeln("before ex");
        fun();
        writeln("after ex");
    }     
    catch(Exception)
    {  
        writeln("catched");
    }
    writeln("hoho\n");
}
```